docker/Dockerfile.flux

```diff
@@ -1,8 +1,14 @@
-FROM alpine:3.9
+FROM bitnami/minideb:stretch
 
 WORKDIR /home/flux
 
-RUN apk add --no-cache openssh ca-certificates tini 'git>=2.12.0' 'gnutls>=3.6.7' gnupg
+RUN install_packages \
```

Comment:
The last commit in https://github.com/bitnami/minideb is from September last year :S

Are we sure it doesn't have any security problems? Shouldn't we do some sort of upgrade before installing new image? It's hard to believe no critical patches are needed since September 2018.

Reply:
Their images are build by an external system and updated regularly, see Docker Hub.

Reply:
@2opremio this image is pushed by Bitnami to Docker Hub every 24h including all security patches

Reply:
The updating of the image is left to the Dockerfile build, I think

Reply:
That's great. I didn't know this was a mutable tag.

BTW, since mutable tags suck. Are we sure only security patches are included when the image is modified?

Reply:
Oh it seems @squaremo [already brought this up](https://github.com/weaveworks/flux/pull/2055#issuecomment-493112068), resolving.